### DISCUSSION TITLE
Recent languages description says 'NaN' when no commits found

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#1026] [Q&A] [answered]
I havent made any public commits recently so my language stats say this:
<img width="219" alt="image" src="https://user-images.githubusercontent.com/42429413/166169520-ca15e220-3f3f-489b-a6ad-4ed2fdf09132.png">
If the day count is NaN then this should just read "No recent push activity found" instead.

**Comment by lowlighter** [comment, marked as answer]
Should have been fixed by #1038
